src/platforms/common/user-feedback/configuration.mdx

```diff
@@ -0,0 +1,276 @@
+
+By default the Feedback integration will attempt to fill in the name/email fields if you have set a user context via [`Sentry.setUser`](https://docs.sentry.io/platforms/javascript/enriching-events/identify-user/). By default it expects the email and name fields to be `email` and `username`. Below is an example configuration with non-default user fields.
+
+```javascript
+Sentry.setUser({
+  email: "foo@example.com",
+  fullName: "Jane Doe",
+});
+
+new Feedback({
+  useSentryUser: {
+    email: "email",
+    name: "fullName",
+  },
+});
+```
+
+### Text Customization
+
+Most text that you see in the default Feedback widget can be customized.
+
+| key                  | default                                | description                                                        |
+| -------------------- | -------------------------------------- | ------------------------------------------------------------------ |
+| `buttonLabel`        | `Report a Bug`                         | The label of the injected button.                                  |
+| `submitButtonLabel`  | `Send Bug Report`                      | The label of the submit button used in the feedback form.          |
+| `cancelButtonLabel`  | `Cancel`                               | The label of the cancel button used in the feedback form.          |
+| `formTitle`          | `Report a Bug`                         | The title at the top of the feedback form.                         |
+| `nameLabel`          | `Name`                                 | The label of the name input field.                                 |
+| `namePlaceholder`    | `Your Name`                            | The placeholder for the name input field.                          |
+| `emailLabel`         | `Email`                                | The label of the email input field.                                |
+| `emailPlaceholder`   | `your.email@example.org`               | The placeholder for the email input field.                         |
+| `messageLabel`       | `Description`                          | The label for the feedback description input field.                |
+| `messagePlaceholder` | `What's the bug? What did you expect?` | The placeholder for the feedback description input field.          |
+| `successMessageText` | `Thank you for your report!`           | The message to be displayed after a succesful feedback submission. |
+
+Example of customization
+
+```javascript
+new Feedback({
+  buttonLabel: "Feedback",
+  submitButtonLabel: "Send Feedback",
+  formTitle: "Send Feedback",
+});
+```
+
+### Theme Customization
+
+Colors can be customized via the Feedback class constructor or by defining CSS variables on the injected button. If you use the default button, it will have an `id="sentry-feedback`, meaning you can use the `#sentry-feedback` selector to define CSS variables to override.
```

Comment:
```suggestion
Colors can be customized via the Feedback class constructor or by defining CSS variables on the injected button. If you use the default button it will have a property `id="sentry-feedback`, meaning you can use the `#sentry-feedback` selector to define CSS variables to override.
```